libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1012,6 +1014,20 @@ void AP_AHRS_DCM::set_home(const Location &loc)
 {
     _home = loc;
     _home.options = 0;
+
+    // log ahrs home and ekf origin dataflash
+    Log_Write_Home_And_Origin();
+
+    // send new home and ekf origin to GCS
+    gcs().send_home();
+    gcs().send_ekf_origin();
+
+    // send text of home position to ground stations
+    gcs().send_text(MAV_SEVERITY_INFO, "Set HOME to %.6f %.6f at %.2fm",
```

Comment:
I strongly don't like this. It's arguably reasonable for a MAVLink command, it's wholly unreasonable to spam it at a GCS every 5 seconds on plane. Given that we have real MAVLink messages for the home position and acking. I'd rather just remove this status text entirely. Either way sending this every 5 seconds on plane is a no go.

Reply:
That's the disarmed case?

Fair call - this is WIP for a reason.  I was interested to see if people thought doing all of this  stuff in AHRS was appropriate at all!

Reply:
Correct. I think sending the MAVLink message is absolutely correct and I'm happy to see that move up, it's just the status text I take issues with.